docs/extensions.adoc

```diff
@@ -495,11 +495,16 @@ per JVM, keep in mind that Spock cannot enforce this.
 
 NOTE: This will acquire a `READ_WRITE` lock for `Resources.SYSTEM_PROPERTIES`, see <<parallel-execution.adoc#parallel-execution, Parallel Execution>> for more information.
 
+[[AutoAttach]]
```

Comment:
If we add an explicit link for a pre-existing section we should use the generated one, to keep old links functioning.
`_autoattach`

Reply:
I did not add this line, but I can remove it, while I am dealing with the other review findings. I am just unsure what you mean by `_autoattach`. Is that meant to be a naming suggestion or another kind of hint? For me, that is a bit too terse.

Reply:
I guess, if you do not supply a label like `[[AutoAttach]]` [Asciidoctor](https://asciidoctor.org/) defines a default label named by the Headline which then would be `_autoattach`.
And @leonard84 meant I could have broken links to the old auto-generated label.